RNTester/Podfile.lock

```diff
@@ -178,8 +182,6 @@ EXTERNAL SOURCES:
     :path: "../Libraries/NativeAnimation"
   React-RCTBlob:
     :path: "../Libraries/Blob"
-  React-RCTCameraRoll:
```

Comment:
What's the reason for deleting the RCTCameraRoll dependency?

Reply:
Good catch! Looks like [it was removed](https://github.com/microsoft/react-native/commit/1adbcfd785cc536cd3f7ee39719eaf663854f258) from the Podfile because it is a deprecated component, but the lock file wasn’t updated (or at least not checked in). The [associated PR](https://github.com/microsoft/react-native/pull/162), however, doesn’t seem to provide any context for why that was necessary.

I can confirm that adding it back to the Podfile makes it work in the test app again, so shall I do that for the time being? I’m sure it will get removed again once we sync with upstream.

Reply:
Awesome. We can *probably* take it out here. I don't know of any clients that were dependent on it, but if there is one then we should just start pulling in the React-Native-Community version of it anyway.

Reply:
Verified no one uses it. Looks good!

Reply:
I’ve added it back for now, upstream FB didn’t get rid of it until 0.61.0